rebasehelper/cli.py

```diff
@@ -143,6 +150,13 @@ def add_args(self):
             help="Specify number of retries in case build fails.",
             type=int
         )
+        self.parser.add_argument(
+            "--enable-option",
```

Comment:
please rename it to indicate that this is intended only for extra arguments for builders. I already provided you with some examples.